src/juicebox/tools/clt/old/StatisticsScript.java

```diff
@@ -0,0 +1,593 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2011-2020 Broad Institute, Aiden Lab
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ *  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ *  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ *  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ *  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ *  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ *  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ *  THE SOFTWARE.
+ */
+
+package juicebox.tools.clt.old;
+
+import juicebox.tools.clt.CommandLineParser;
+import juicebox.tools.clt.JuiceboxCLT;
+
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.text.NumberFormat;
+import java.util.*;
+
+public class StatisticsScript extends JuiceboxCLT {
+    //variables for getting parameters of input file, flags set to default initially
+    String outFile;
```

Comment:
set permission level for all these parameters; they should probably all be private. some should also be final (the ones that are defined only once and should not change)